Copy edit and suggestions for the writing handbook (#1082)
GoogleChrome/web.dev: src/site/content/en/handbook/content-checklist/index.md

> This is a checklist for web.dev content reviews. It includes critical high-level issues and lower-level issues that commonly appear in drafts.

Author: I suggest rephrasing this to be actionable. "Use this checklist to make content reviews go faster" or something. Follow the style guide rule about value statements.

Reviewer: So, I didn't make this imperative because it's really only for a subset of handbook readers--the content reviewers. Maybe it should say that?

Reviewer: Rephrased in Google Doc

> ## Basics

Author: does "piece" mean blog posts/articles/docs? This feels kind of strange and not commonly used. I'm also not sure how it would translate when you're thinking about localization.

Reviewer: Yes, need an umbrella term that includes both posts and codelabs. Wasn't thrilled with _piece_ either. Suggestions welcome.

Reviewer 2: I replaced `piece` with `content`

1. Does the piece provide value for one or more of the web.dev [personas](/handbook/audience)?
1. Do the title, subtitle, and description convey a [value proposition](/handbook/audience/#value-propositions)?

> 1. Is the piece 1,000 words or less? If it's longer, is there a good reason for it to be longer?

Author: what's a good reason?  Is there a decision matrix for exceptions?

Reviewer: I mean, no, haha. I was hoping to keep this more ad hoc?

Reviewer 3: Actually there are good reasons. For some new, large APIs, 1000 words is not sufficient to capture what needs to be said. WebXR Device API, for which I'll be submitting articles in September suffers from this. I plan to address this by proposing a multi-part series. Do we have guidance for such a creature?

> 1. Are all statements and advice accurate and aligned with best practices?

Author: link to best practices? How do contributors know that they're aligned to best practices?

Reviewer: By using their domain knowledge, I was thinking. But I take the point that I think you're making: if that's the assumption, and we're not having the reviewer check against a reference, is this even a valuable question to ask? Maybe we cut it.

Reviewer 3: This is entirely within domain knowledge meaning that the best judges of best practices are often the content contributors themselves, especially in articles about new features. Take service workers for instance. Many early articles were written by Jake Archibald. At that point he was the only source of best practices because he invented the thing.

One of the reasons we created WebFundamentals was that current best practices aren't written down yet. That's because best practices are often moving targets with the web. (I hope you'll take that at face value. This response is getting too long.)

This really needs to be (in my opinion) is a reminder to content contributors to put best practices in writing and to exemplify best practices in their code examples.

Reviewer 2: I agree that this one is iffy but not sure how to make better, so gonna put off for now

## Instruction
1. If the piece expects readers to have prior knowledge, is that made clear up front?
1. Does the piece explain why the process being taught matters?

> 1. Do the examples support all key claims and ideas?

Reviewer: ```suggestion
1. Are there examples to support all key claims and ideas?
```
The proposed edit changed the meaning a bit. I want to ask whether there are enough examples, not whether the examples provided are doing their job.

1. Are instructions written in second person (_you/your_), not first person (_we/our_)?

> 1. Do the common instructions follow the web.dev [Instruction components](/handbook/web-dev-components/#instruction)?

Reviewer: ```suggestion
1. Are common instructions included using the web.dev [Instruction components](/handbook/web-dev-components/#instruction)?
```
Again, the meaning was changed. Common instructions aren't following a template; rather writers should use a shortcode so the instructions are populated by the component.

## Media

> 1. Do the images and videos clarify ideas that would be difficult to understand from text alone?

Reviewer: Similar to the examples question above, this changes the meaning. I'd prefer to revert so it's clear we're asking whether people are including images and video where they're needed--though improved phrasing welcome!

Reviewer 3: I agree with Michael on this. Someone actually violated item two recently. You wouldn't think we'd need to say it, but alas.

> 1. Are all links to web.dev pages and assets relative?

Author: Link or explanation to what it means to be relative?

Reviewer 2: Will link to this: https://developer.mozilla.org/en-US/docs/Learn/Common_questions/What_is_a_URL#Examples_of_relative_URLs

## Mechanics
1. Is the text free of spelling and capitalization errors? Check the [word list](/word-list).
1. Are [dashes and hyphens](/handbook/grammar/#dashes-and-hyphens) used correctly?

> 1. Are titles and headings in sentence case with no terminal period?

Author: what's a terminal period? Might be simpler to say "Are they free of punctuation?"

Reviewer: ```suggestion
1. Are titles and headings in sentence case with no period at the end?
```
Well, that's not really accurate. They might need internal punctuation (commas, hyphens), for example. Suggested a revision.

Reviewer: Fixed in Google Doc
